snooty/rstspec.toml

```diff
@@ -1003,6 +1003,10 @@ type = "explicit_title"
 help = """Show a FontAwesome 4 icon."""
 type = "explicit_title"
 
+[role.icon-lg]
+help = """Show a LeafyGreen icon."""
+type = "explicit_title"
+
 [role.rfc]
```

Comment:
This is good. You will need to also open a third PR 😱 :) 
Our rstspec.toml source of truth is actually in another branch of the parser. So, this is fine to have here, but it won't be taken into account. The branch name is `latest`. You can open a PR for that branch that just adds these lines!

Reply:
3rd PR added [here](https://github.com/mongodb/snooty-parser/pull/516)!